woocommerce/google-listings-and-ads
# Add `bundlewatch/bundlewatch` to dev dependencies.

### Changes proposed in this Pull Request:
Add `bundlewatch/bundlewatch` to dev dependencies.
to monitor our progress in reducing bundle size.

Fixes https://github.com/woocommerce/google-listings-and-ads/issues/818.


### Screenshots:
![bundlewatch](https://user-images.githubusercontent.com/17435/123252811-fa33c800-d4ec-11eb-89c3-87d5cdbf22d1.gif)



### Detailed test instructions:

1. check out this branch
2. `npm install`
3. `npx bundlewatch`
4. Click on the link for the detailed report

### To be done:
- [ ] Authorize the app, to enable GitHub check. https://service.bundlewatch.io/setup-github


### Changelog entry

> Tweak - Add bundlewatch to monitor bundle size in GitHub PRs.
